nsxt/utils_test.go

```diff
@@ -719,6 +719,40 @@ func testAccNsxtPolicyMultitenancyContext() string {
 	return ""
 }
 
+func testAccNsxtMultitenancyContext(includeVpc bool) string {
```

Comment:
Maybe it's a good idea to abandon `testAccNsxtPolicyMultitenancyContext` entirely and replace it with this (obviously not in this PR's scope).